scripts/install/opt/seagate/cortx/motr/config/be_resize.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+# set -x
+
+current_time=$(date +"%Y_%m_%d__%H_%M_%S")
+LOG_DIR="/var/log/seagate/motr"
+mkdir -p $LOG_DIR
+LOG_FILE="$LOG_DIR/be_log_resize_$current_time.log"
+touch $LOG_FILE
+MOTR_CONF_FILE="/etc/sysconfig/motr"
+M0BETOOL="m0betool"
+MD_DIR="/var/motr"
+BE_REPLY="be_recovery_run"
+BE_RESIZE="be_log_resize"
+
+local_md_volume=""
+remote_md_volume=""
+new_be_log_size=""
+current_be_log_size=""
+local_node=""
+remote_node=""
+loca_ios_fid=""
+remote_ios_fid=""
+
+be_resize_log() {
+    echo "[`date`]: $@" 2>&1 | tee -a ${LOG_FILE}
+}
+
+be_resize_soft_exit() { be_resize_log "$@" ; exit 0; }
+
+be_resize_hard_exit() { be_resize_log "$@" ; exit 1; }
+
+# Verifying cluster status
+check_cluster_status() {
+	if  [[ $((hctl status) 2>&1 | grep "Cluster is not running") ]]; then
```

Comment:
Check for m0d's too, which should not be running.

Reply:
Also for LDR R1, if mount point exists, return from here.

Reply:
Handled two conditions.